recipe/meta.yaml

```diff
@@ -24,9 +24,9 @@ requirements:
     - libgomp      # [linux]
     - ninja
   host:
-    - kokkos >=4.0.0,<5
+    - kokkos >=4.2.0,<5
```

Comment:
Here, I am not sure about the minimum compatible version of Kokkos with KokkosKernels 4.2.01.

Reply:
I'd err on the side of caution since they always release in pair at the moment. So let's pin to 4.2.01 unless there is a reason not to?

Reply:
I agree.